src/envoy/http/service_control/handler_impl.h

```diff
@@ -117,6 +117,9 @@ class ServiceControlHandlerImpl
   std::string uuid_;
   std::string api_key_;
 
+  // The size of request headers.
+  int request_header_size_;
```

Comment:
add comment on 
// If request header is modified, the original downstream header should be used as request_header_size
// this variable is used to remember the header size when HandlerImpl object is created.

Reply:
done